mods/ra/maps/soviet-09/soviet09.lua

```diff
@@ -0,0 +1,109 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+TruckStops = { TruckStop1, TruckStop2, TruckStop3, TruckStop4, TruckStop5, TruckStop6, TruckStop7, TruckStop8 }
+MissionStartAttackUnits = { StartAttack1tnk1, StartAttack1tnk2, StartAttackArty1, StartAttackArty2, StartAttackArty3 }
+TruckEscape = { TruckEscape1, TruckEscape2, TruckEscape3, TruckEscape4, TruckEscape5, TruckEscapeWest }
+BackupRoute = { TruckEscape2, TruckEscape1, TruckEscapeEast }
+
+MissionStart = function()
+	Utils.Do(TruckStops, function(waypoint)
+		StolenTruck.Move(waypoint.Location)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(5), function()
+		Utils.Do(MissionStartAttackUnits, function(actor)
+			actor.AttackMove(DefaultCameraPosition.Location)
+		end)
+	end)
+
+	Trigger.AfterDelay(DateTime.Seconds(45), function()
+		Media.DisplayMessage("Commander, the truck has stopped at a nearby allied base.\nAllied radio intercepts say the truck has orders to flee the battlefield\nif any Soviet units approach the base.")
+	end)
+
+	Trigger.OnKilled(StolenTruck, function()
+		USSR.MarkCompletedObjective(DestroyTruck)
+		USSR.MarkCompletedObjective(DefendCommand)
+	end)
+
+	Trigger.OnKilled(CommandCenter, function()
+		USSR.MarkFailedObjective(DefendCommand)
+	end)
+end
+
+Trigger.OnEnteredProximityTrigger(TruckAlarm.CenterPosition, WDist.FromCells(13), function(actor, triggerflee)
```

Comment:
I would shrink this a bit? In the originals the trigger fired when you were near the sandbags, but we're further away. Ending clip for reference: https://www.twitch.tv/hambtv/clip/DifferentBovineParrotCopyThis

Reply:
Updated, I moved the waypoint up by one and reduced the distance by two. I can't match it up perfectly without getting in too close from the sides. I want a proximity trigger instead of footprint here to account for paratroopers.

Reply:
Makes sense, sounds good